native/src/components/FeedbackContainer.tsx

```diff
@@ -1,17 +1,33 @@
 import React, { ReactElement, useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
 import styled from 'styled-components/native'
 
 import { SEND_FEEDBACK_SIGNAL_NAME } from 'shared'
 import { createFeedbackEndpoint, FeedbackRouteType } from 'shared/api'
+import { config } from 'translations'
 
+import buildConfig from '../constants/buildConfig'
 import { determineApiUrl } from '../utils/helpers'
 import sendTrackingSignal from '../utils/sendTrackingSignal'
 import { reportError } from '../utils/sentry'
 import Feedback from './Feedback'
+import Text from './base/Text'
+import TextButton from './base/TextButton'
 
 const Container = styled.View`
   flex: 1;
   background-color: ${props => props.theme.colors.backgroundColor};
+  padding: 8px 20px;
+  gap: 8px;
+`
+
+const BoldText = styled(Text)`
+  font-weight: 600;
+`
+
+const TextBeforeButton = styled(BoldText)`
+  margin-top: 8px;
+  text-align: center;
```

Comment:
:wrench: See comments for web, I think we should avoid naming things based on the style/position they use as they can easily change. Instead, they should focus more on functionality/usage structure imo.